### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Kris2ofer**

### CODE REVIEW

File: lesson4/index.md [hunk 1]
```diff
@@ -36,11 +36,23 @@ HTML tags are...
...
 (please give related terms and vocabulary that you taught during the class)
 
-- `html tag` - An html tag is
-- `hypertext` - Hypertext means this and that
-- `...` - ...
 
----
+- Empty HTML Elements - These are tags with no content 
+- Headings tag (h1-h6)- Represents a container for introductory content or a set of navigational links. Used by search engines.
+- <pre> tag - This defines preformatted text. It helps to preserve the white space inside your html element
```
Comment (angelsvirkov) [inline lesson4/index.md:42]: @impactmass, @otaruMendez ,
Do you think we should put html tags descriptions inside the Glossary? Or should we put them in another section or should we leave them out in general? I am thinking about how to make a consistent approach for all lecture notes. As adding tags info here is a lot of work

Comment (otaruMendez) [reply lesson4/index.md:42]: I would vote for leaving it out, it can be quite challenging maintaining this structure for subsequent lessons

Comment (impactmass) [reply lesson4/index.md:42]: Let's leave Tag Definitions out of the Glossary (in general). I agree with the reason Babatunde gave. 

@Kris2ofer, if needed you can link to [MDN's HTML elements reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) in the section where you explain these tags.

**Comment by angelsvirkov** [comment]
Closing this pull request because we have already adjusted lesson 4 notes.

**Pull request closed without merging**